org-roam-macs.el

```diff
@@ -56,7 +58,10 @@ to look.
                         (error-message-string err)
                         ,templates))))
 
-
+(defmacro org-roam-message (msg)
+  "Message MSG when `org-roam-verbose' is true."
+  `(when org-roam-verbose
+     (message msg)))
```

Comment:
I think you need to expand `msg` with `,` here.

Reply:
i made that change locally, not sure why it didn't show up here!